_posts/con-commands/2020-05-01-mom_zone_zoomout.md

```diff
@@ -0,0 +1,10 @@
+---
+title: mom_zone_zoomout
+category: command
+tags:
+  - zones
+  - crosshair
+cvar_ref: mom_zone_grid
+---
+
+Increases the maximum distance of the zoning reticle. The amount increased is equivalent to the grid size (see [`{{ page.cvar_ref }}`](/var/{{ page.cvar_ref }})).
```

Comment:
Unrelated to review: could be good to make this a static element (like the min/max/default sections) on pages, saving you from having to do the `(see [`{{ page.cvar_ref }}`](/var/{{ page.cvar_ref }}))` everywhere.